src/main/java/cloud/cholewa/game/logic/GameStarter.java

```diff
@@ -0,0 +1,41 @@
+package cloud.cholewa.game.logic;
+
+import cloud.cholewa.game.Player;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class GameStarter {
+
+    public static int setNumberOfPlayers(Scanner scanner) {
+        int number;
+
+        System.out.print("Podaj liczbę graczy (2-4): ");
+        number = scanner.nextInt();
+
+        if (number < 2 || number > 4) {
+            System.out.println("Błędna ilość graczy !");
+            System.exit(0);
```

Comment:
System.exit to raczej kiepska praktyka, lepiej rzucić wyjątek, obsłużyć w GameEngine i zakończyć program

Reply:
Poprawione.... mam nadzieję, że dobrze zrozumiałem Twoją podpowiedź.